app/models/manager_refresh/inventory_collection/builder/container_manager.rb

```diff
@@ -0,0 +1,237 @@
+module ManagerRefresh
+  class InventoryCollection
+    class Builder
+      class ContainerManager < ::ManagerRefresh::InventoryCollection::Builder
+        # TODO: (agrare) Targeted refreshes will require adjusting the associations / arels. (duh)
```

Comment:
@agrare I wonder what you've meant by this :-)

Reply:
Ha I'm pretty sure I didn't write this, let me track down where it came from

Reply:
Looks like it was @cben, I took his changes from https://github.com/ManageIQ/manageiq/pull/14337 and applied to the new repo after it was split out here https://github.com/ManageIQ/manageiq-providers-kubernetes/pull/30

Link to the original comment: https://github.com/ManageIQ/manageiq/pull/14337/files#diff-65533e4742c2672e65b884331a5ad673R43

Reply:
@agrare sorry I looked only to annotation, I'll change it